src/handlers/createHandler.tsx

```diff
@@ -469,41 +469,49 @@ export default function createHandler<
 
       this.propsRef.current = events;
 
-      const child: any = React.Children.only(this.props.children);
-      let grandChildren = child.props.children;
-      if (
-        __DEV__ &&
-        child.type &&
-        (child.type === 'RNGestureHandlerButton' ||
-          child.type.name === 'View' ||
-          child.type.displayName === 'View')
-      ) {
-        grandChildren = React.Children.toArray(grandChildren);
-        grandChildren.push(
-          <PressabilityDebugView
-            key="pressabilityDebugView"
-            color="mediumspringgreen"
-            hitSlop={child.props.hitSlop}
-          />
+      try {
```

Comment:
I wonder if it is possible to convert this enormous try/catch into something like
```ts
let child: any; // TODO: fix type
try {
  child = React.Children.only(this.props.children);
} catch (e) {
  throw new Error(...);
}
```
or in functional style:
```ts
let child: any = (() => {
  try {
    return React.Children.only(this.props.children);
  } catch (e) {
    throw new Error(...);
  }
})();
```
Obviously, the first variant has better performance.

Thanks to this change we won't get weird error messages if some other part of the code fails.

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/2131/commits/8933f4eb7bea0608e6eacc2775184c73bcfb3285.